FirstProject/Program.cs

```diff
@@ -1 +1,40 @@
-﻿
\ No newline at end of file
+﻿using FirstProject;
+using System.ComponentModel.Design;
+
+Console.WriteLine("FirstProject");
+
+
+    BasedCalculator basedcalculator = new BasedCalculator();
+Console.WriteLine("Kalkulator podstawowy");
+Console.WriteLine("Dodawanie - wybierz 1");
+Console.WriteLine("Odejmowanie - wybierz 2");
+Console.WriteLine("Mnożenie - wybierz 3");
+Console.WriteLine("Dzielenie - wybierz 4");
+int action = Convert.ToInt32(Console.ReadLine());
+    Console.WriteLine("Podaj liczby");
+    int x = Convert.ToInt32(Console.ReadLine());
```

Comment:
Raczej jestem przeciwny dawaniu wartości x i y, bo może się zdarzyć ze nowa osoba nie bedzie wiedziała o co chodzi po nazwie, wolę np value1, value2, albo number1, number2 czy cos w tym stylu - coś aby bardziej wskazywało na to że to jest wartość liczbowa na której działamy